crypto/provider_core.c

```diff
@@ -436,9 +452,9 @@ static int provider_activate(OSSL_PROVIDER *prov)
     return 1;
 }
 
-int ossl_provider_activate(OSSL_PROVIDER *prov)
+int ossl_provider_activate(OSSL_PROVIDER *prov, OPENSSL_CTX *libctx)
```

Comment:
Do you actually use this anywhere?  I can't see it...  I would suggest that *this* function should simply pass NULL as second argument to `provider_activate`

Reply:
Ok, having looked around (especially at what `fipsprov.c` does, I don't see the necessity to change `ossl_provider_activate`.  I understand the `provider_activate(prov, ctx)` call in `ossl_provider_forall_loaded`, and unless I'm missing something, that's the only place where this is needed to cover for the FIPS module's need.

Reply:
Ok, having looked around (especially at what `fipsprov.c` does, I don't see the necessity to change `ossl_provider_activate`.  I understand the `provider_activate(prov, ctx)` call in `ossl_provider_forall_loaded`, and unless I'm missing something, that's the only place where this is needed to cover for the FIPS module's need.